pkgs/by-name/ti/tint/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "tint";
+  version = "0.1.6";
+
+  src = fetchFromGitHub {
+    owner = "ashish0kumar";
+    repo = "tint";
+    tag = "v${version}";
+    hash = "sha256-TZYAcs6h4Fv0XpUqzgCcAF5cHGaVKMMCJ6MTAH6C6Jo=";
+  };
+
+  vendorHash = null;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+  ];
+
```

See if you can add these two lines:

```suggestion

  doInstallCheck = true;
  nativeInstallCheckInputs = [
    versionCheckHook
  ];

```

https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook